srml/support/procedural/src/storage/transfo.rs

```diff
@@ -0,0 +1,619 @@
+// Copyright 2017-2018 Parity Technologies (UK) Ltd.
```

Comment:
I think the file should be called `transformation`. `transfo` is not very expressive.